### DISCUSSION TITLE
配置文件高亮显示不全

### DISCUSSION THREAD

**Discussion by likecutie** [1Panel-dev/1Panel#5455] [Q&A]
### 1Panel 版本

1.5.5

### 请描述您的需求或者改进建议

如图，在include所在行*号后面的配置都是灰色的
![图片](https://github.com/1Panel-dev/1Panel/assets/49803345/2f8fa3ba-78de-4450-ab96-91127d2b69a1)
删除*号高亮正常
![图片](https://github.com/1Panel-dev/1Panel/assets/49803345/5819855e-4b89-4b03-93a1-1eb437dc4f5c)

**Comment by likecutie** [comment]
上面的文字描述中两个星号被吞了，变成斜体了。

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，我们会在后续版本优化上述问题。

**Comment by ssongliu** [comment]
目前配置使用的是 vue-codemirror，支持高亮的语言库不包括 nginx

https://github.com/orgs/codemirror/repositories?q=lang-&type=all

**Reply by likecutie** [reply]
原谅我迟到的回复，查询资料后得知，legacy-modes 支持 nginx，且能配置到 vue-codemirror
https://github.com/codemirror/legacy-modes
https://github.com/sweepai/sweep/commit/8a60881fe058e207f493d0f622b2c3909e6c5c23
